docs/k8s-quickstart.asciidoc

```diff
@@ -12,9 +12,12 @@ Eager to get started? This quick guide shows you how to:
 * <<{p}-persistent-storage,Use persistent storage>>
 * <<{p}-check-samples,Check out the samples>>
 
-**Requirements**
+**Supported versions**
 
-Make sure that you have link:https://kubernetes.io/docs/tasks/tools/install-kubectl/[kubectl] version 1.11+ installed.
+Before you start, make sure that you have the following versions installed:
```

Comment:
I don't think we need this line because the title explains it all.

Also, it's misleading because it implies that Elasticsearch should already be installed to use the operator.

Reply:
Thank you, comment integrated.